enterprise/frontend/src/metabase-enterprise/metabot/metabot.unit.spec.tsx

```diff
@@ -16,7 +16,13 @@ import {
 import { logout } from "metabase/auth/actions";
 import * as domModule from "metabase/lib/dom";
 import { useRegisterMetabotContextProvider } from "metabase/metabot";
-import type { SuggestedMetabotPrompt, User } from "metabase-types/api";
+import {
```

Comment:
The changes were made as follows:
- brought over any tests in `metabot.unit.spec.tsx` to `metabot-streaming.unit.spec.tsx` that didn't already exist
- fixed those tests so they worked with streaming
- renamed `metabot-streaming.unit.spec.tsx` to `metabot.unit.spec.tsx`

Everything should be previously approved changes. Now that this file is so long I've created a task to split up the unit tests here: [BOT-315](https://linear.app/metabase/issue/BOT-315/split-up-metabot-fe-unit-tests)